lib/bemxjst/index.js

```diff
@@ -273,11 +275,16 @@ BEMXJST.prototype.runEmpty = function runEmpty() {
   return '';
 };
 
+BEMXJST.prototype.runUnescape = function runUnescape(context) {
+  this.context._listLength--;
+  return '' + context;
+};
+
 BEMXJST.prototype.runSimple = function runSimple(context) {
   this.context._listLength--;
   var res = '';
   if (context && context !== true || context === 0)
-    res += context;
+    res += this.context.escapeContent ? utils.xmlEscape(context) : context;
```

Comment:
Please add `this.escapeContent = false;` to the `Context`. The property should always be present.